### PR TITLE
feat(devicefarm): maintain device listing in a centralized location (RA)

### PR DESCRIPTION
**Pull request by Nintendot**
## Overview

All details in: https://github.com/telusdigital/telus-isomorphic-starter-kit/pull/167

#### Meta

- [x] provide a descriptive topic
- [x] provide an overview of contribution
- [x] no sensitive content included, such as:
  - security & privacy policy violating content
  - content considered competitive intelligence
  - keys, tokens or credentials
- [x] documentation format follows [this template][template]
- [x] fork is up to date ([see this guide from github](https://help.github.com/articles/syncing-a-fork/))
- [x] "work in progress" commits are squashed ([see "Squashing Commits"](https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History))
- [x] commits follow the [Karma][karma-format] or [Commitizen](https://www.npmjs.com/package/commitizen) format

[template]: ../.template.md
[karma-format]: https://karma-runner.github.io/1.0/dev/git-commit-msg.html

### CODE REVIEW

**Comment by crimeminister** [comment]
Having this QA device list available from a central location is a good idea. There should probably be some explanation of the file contents and purpose elsewhere in the RA; perhaps with some helpful getting started tutorial text, e.g. commands to run to fetch and use the device list.

**Comment by Nintendot** [comment]
@crimeminister yes you are right, even though the starter-kit mentions it, it makes sense to explain the purpose in RA itself, I will get it updated

File: testing/tools_platforms/devicefarming.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+
+## What
+
+A onboarding guide for teams to start running their [e2e tests][e2e tests]
+
+
+## How
+
+Devicefarm has been integrated in the [TELUS Digital starter-kit][sk] for running [Nightwatch][nw] tests.
```
Comment (crimeminister) [inline testing/tools_platforms/devicefarming.md]: The Nightwatch link isn't accessible for me.

Comment (Nintendot) [reply testing/tools_platforms/devicefarming.md]: Nice catch, and thank you sir for fixing it

**Pull request merged**